style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
Great job @Osoro254Alex 🤝

- [ ] [Optional] As specified in the project requirements, kindly use a class to style your `h1`.